docs/source/jupyter_notebook.rst

```diff
@@ -16,6 +12,78 @@ To create a Jupyter Notebook, run:
 
 .. code-block:: console
 
-   llmo workspace notebooks create --name my-test-notebook
+   llmo workspace notebooks create my-notebook
+
+By default, there is no GPU is allocated to the Jupyter Notebook.
+If you want to allocate a GPU to the Jupyter Notebook, run:
+
+.. code-block:: console
+
+   llmo workspace notebooks create my-gpu-notebook --gpu 1
+
+There are other options that you can specify when creating a Jupyter Notebook, such as environment.
+You can see the list of options by using the `--help` flag.
+
+Once the Jupyter Notebook is created, you can access it by running:
+
+.. code-block:: console
+
+   # Open the Jupyter Notebook in your browser
+   llmo workspace notebooks open my-notebook
+
+
+Stopping and Restarting a Jupyter Notebook
+------------------------------------------
+
+To stop a Jupyter Notebook, run:
+
+.. code-block:: console
+
+   llmo workspace notebooks stop my-notebook
+
+To restart a Jupyter Notebook, run:
+
+.. code-block:: console
+
+   llmo workspace notebooks start my-notebook
+
+You can check the current status of the Jupyter Notebook by running:
+
+.. code-block:: console
+
+   llmo workspace notebooks list
+   llmo workspace notebooks get my-notebook
+
+
+OpenAI API Integration
+----------------------
+
+Jupyter Notebook can be integrated with OpenAI API. Created Jupyter Notebook is pre-configured with OpenAI API URL and API key.
+All you need to do is to install the `openai` package.
```

Comment:
Could you add a note/caution somewhere in this page and mention that the token passed to `OpenAI()` is JWT by default and it can expire? Once it expires, a user can pass an API key.